### ISSUE TITLE
Broken assertion

### ISSUE THREAD

**Issue by jwindhager** [BodenmillerGroup/imctools#110]
Just noticed along the way that this line won't do what you intended it to do:

https://github.com/BodenmillerGroup/imctools/blob/e546bd3e452a1fc8f6be53c45f37d13f5c960722/imctools/converters/ome2analysis.py#L40

**Comment by plankter** [comment]
@jwindhager , thank you for the report. True, it should be something like

```
selected_set = set(selected.unique())
selected_set.discard(0)
selected_set.discard(1)
assert len(selected_set) == 0, f"Values in 'usedcolumn' column should contain only 0/1"
```

I'll prepare an update tomorrow.

**Comment by jwindhager** [comment]
Thanks! Or simply:

    selected.isin((0, 1)).all()

Btw, while you're at it, you could replace the line following the assertion with a simple `.loc`:

    metals = panel.loc[selected == 1, metalcolumn].tolist()

Just my 2c and suggestions - pandas is a powerful beast :-)

**Comment by plankter** [comment]
Fixed in v 2.1.7
